mantidproject/mantid
# Publish test results from the new Jenkins pipeline

*Please ensure #33803 is merged first.*

**Description of work.**

Add post actions to the Jenkins deployment pipeline to publish test results to the main job page.

**To test:**

Review the code changes. The same changes were made in a [testing job](https://builds.mantidproject.org/job/testing-pipeline/33/) using [this test jenkinsfile](https://github.com/martyngigg/sandbox/blob/main/jenkins/build.jenkinsfile). See that they do the same thing.

*There is no associated issue.*

*This does not require release notes* because **it is an internal change.**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
